templates/zerver/help/edit-profile.md

```diff
@@ -0,0 +1,20 @@
+# Edit your profile
+Editing your profile lets you adjust the details of your account and gives a better description of who you are to your team members. The following instructions show you how to change your name, choose an avatar, select a default language, and manage notifications.
+
+### Finding the settings page
+1. Click the cog(![cog](/static/images/help/cog.png)) icon in the top right corner.
+2. From the dropdown menu, choose the "**Settings**" option. 
+3. This will replace the middle column, the messages column with the "**Settings**" page.
```

Comment:
Add a comma after the second column, as follows: `...the messages column, with...`